pkg/operator/controller/garden/reconciler_delete.go

```diff
@@ -115,6 +116,10 @@ func (r *Reconciler) delete(
 			Name: "Destroying Kube State Metrics",
 			Fn:   component.OpDestroyAndWait(kubeStateMetrics).Destroy,
 		})
+		destroyVirtualGardenGardenerAccess = g.Add(flow.Task{
+			Name: "Destroying Gardener virtual garden access resources",
+			Fn:   virtualGardenGardenerAccess.Destroy,
+		})
 		destroyVirtualGardenGardenerResourceManager = g.Add(flow.Task{
```

Comment:
Don't we have to wait for the `destroyVirtualGardenGardenerAccess` task before starting to destroy the GRM for the virtual cluster? Otherwise, there could be racy behaviour where this VG-GRM cannot delete the `ManagedResource` for the gardener-access?